### PR TITLE
G Suite service acct consent screen

### PR DESCRIPTION
**Pull request by Tabrarian**
Adding a screenshot and text for configuring the OAuth consent screen when creating a service account for G Suite Groups.

### CODE REVIEW

File: _docs/howto/configure/gsuite-group-collection.md [hunk 1]
```diff
@@ -19,16 +19,23 @@ service account just for this functionality:
...
+            to display on the consent screen, then click **Create**. To change the product name or add
+            details to the consent screen later, edit your
+            [OAuth consent screen](https://console.developers.google.com/apis/credentials/consent) settings.
+            
+            <img src="../../images/howto/create-service-account.png" alt="create service account window with product name field highlighted">
         1.  Click **Create**.
     1.  To create and download a JSON key for the service account:
         1.  Click **More** on the service account row, then click **Create key**.
-        <img src="../../images/howto/create-key.png" alt="more menu with create key highlighted">
+            <img src="../../images/howto/create-key.png" alt="more menu with create key highlighted">
```
Comment (mirons-google) [inline _docs/howto/configure/gsuite-group-collection.md]: Actually the better way to do this (in markdown and not HTML) is to use this [syntax](https://guides.github.com/features/mastering-markdown/), e.g.

`[Documentation Categories Screenshot](#)`

Please fix throughout.

File: _docs/howto/configure/gsuite-group-collection.md [hunk 2]
```diff
@@ -38,8 +45,8 @@ service account just for this functionality:
         ```
         https://www.googleapis.com/auth/admin.directory.group.readonly
         ```
-    1. Click **Authorize**.
-    <img src="../../images/howto/admin-security.png" alt="manage api client access in Google Admin Security settings">
+    1.  Click **Authorize**.
```
Comment (mirons-google) [inline _docs/howto/configure/gsuite-group-collection.md]: Did you intend to add the space before "Click"?

Comment (Tabrarian) [inline _docs/howto/configure/gsuite-group-collection.md]: Yes; it now matches the indentation of every other same-level item on the page.

File: _docs/howto/configure/gsuite-group-collection.md (hunk 1 above)
Comment (Tabrarian) [inline _docs/howto/configure/gsuite-group-collection.md]: Done, and thanks; when I originally searched for how to do this, I had only found the HTML reference.

**Comment by mirons-google** [comment]
Oops, it appears htmlproofer doesn't like the '#' links. Can you see what happens if you don't do the `(#)` piece?

**Comment by Tabrarian** [comment]
Without the # links, it renders the outside brackets; I removed those as well, so the formatting matches the syntax on the other page you linked. It appears correct on the UI preview.

**Comment by mirons-google** [comment]
Cool, travis is happy [now](https://travis-ci.org/GoogleCloudPlatform/forseti-security/builds/261113961?utm_source=github_status&utm_medium=notification)!

**Pull request merged**